# RNA Seq Analysis Papers

<ul>

                             

 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(1).pdf" style="text-decoration:none;">A reanalysis of mouse ENCODE comparative gene expression data [version 1; peer review: 3 approved, 1 approved with reservations]</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(2).pdf" style="text-decoration:none;">RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR [version 1; peer review: 3 approved]</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(3).pdf" style="text-decoration:none;">Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(4).pdf" style="text-decoration:none;">Selecting Between-Sample RNA-Seq Normalization Methods from the Perspective of their Assumptions</a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(5).pdf" style="text-decoration:none;">S-IRFindeR: stable and accurate measurement of intron retention</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(6).pdf" style="text-decoration:none;">Curation of over 10,000 transcriptomic studies to enable data reuse</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(7).pdf" style="text-decoration:none;">Salmon: Accurate, Versatile and Ultrafast Quantification from RNA-seq Data using Lightweight-Alignment</a></li>

 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(8).pdf" style="text-decoration:none;"> On the widespread and critical impact of systematic bias and batch effects in single-cell RNA-Seq data</a></li>
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(9).pdf" style="text-decoration:none;">A Comparison of Methods: Normalizing High-Throughput RNA Sequencing Data</a></li>
  
   
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(10).pdf" style="text-decoration:none;">The impact of amplification on di↵erential expression analyses by RNA-seq </a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(11).pdf" style="text-decoration:none;">Fast and accurate single-cell RNA-Seq analysis by clustering of transcript-compatibility counts</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(12).pdf" style="text-decoration:none;">Dumpster diving in RNA-sequencing to find the source of every last read</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(13).pdf" style="text-decoration:none;">Differential analysis of RNA-Seq incorporating quantification uncertainty</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(14).pdf" style="text-decoration:none;">Batch effects and the effective design of single-cell gene expression studies</a></li>
                              
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(15).pdf" style="text-decoration:none;">Assessment of single cell RNA-seq normalization methods</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(16).pdf" style="text-decoration:none;">CIDR: Ultrafast and accurate clustering through imputation for single cell RNA-Seq data</a></li>

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(17).pdf" style="text-decoration:none;">Power Analysis of Single Cell RNA‐Sequencing Experiments</a></li>   
  
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(18).pdf" style="text-decoration:none;">Overcoming confounding plate effects in differential expression analyses of single-cell RNA-seq data</a></li> 

  
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(19).pdf" style="text-decoration:none;">A framework for RNA quality correction in differential expression analysis</a></li> 

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(20).pdf" style="text-decoration:none;">MetaSRA: normalized sample-specic metadata for the Sequence Read Archive</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(21).pdf" style="text-decoration:none;">Bias, robustness and scalability in differential expression analysis of single-cell RNA-seq data</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(22).pdf" style="text-decoration:none;">SQUID: Transcriptomic Structural Variation Detection from RNA-seq</a></li> 
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(23).pdf" style="text-decoration:none;">Thousands of large-scale RNA sequencing experiments yield a comprehensive new human gene list and reveal extensive transcriptional noise</a></li> 
 

   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(24).pdf" style="text-decoration:none;">A step-by-step workflow for low-level analysis of single-cell RNA-seq data [version 1; peer review: 5 approved with reservations]</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(25).pdf" style="text-decoration:none;">Genome-wide repressive capacity of promoter DNA methylation is revealed through epigenomic manipulation</a></li>                              
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(26).pdf" style="text-decoration:none;">Extracting allelic read counts from 250,000 human sequencing runs in Sequence Read Archive</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(27).pdf" style="text-decoration:none;">RNA Sequencing Data:
Hitchhiker's Guide to Expression Analysis</a></li>
   
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(28).pdf" style="text-decoration:none;">A comprehensive evaluation of
normalization methods for Illumina high-throughput RNA sequencing data analysis</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(29).pdf" style="text-decoration:none;">Gene set analysis approaches for RNA-seq data: performance evaluation and application guideline </a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(30).pdf" style="text-decoration:none;">Comparative evaluation of isoform-level gene expression estimation algorithms for RNA-seq and exon-array platforms</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(31).pdf" style="text-decoration:none;">Comparison of methods to detect differentially expressed genes between single-cell populations</a></li> 
    <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(32).pdf" style="text-decoration:none;">Benchmark analysis of algorithms for determining and quantifying full-length mRNA splice forms from RNA-seq data</a></li> 

   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(33).pdf" style="text-decoration:none;">Swimming downstream: statistical analysis of differential transcript usage following Salmon quantification [version 1; peer review: 3 approved with reservations]</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(34).pdf" style="text-decoration:none;">Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</a></li> 
 
  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(35).pdf" style="text-decoration:none;">Count-based differential expression analysis of RNA sequencing data using R and Bioconductor</a></li> 

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(36).pdf" style="text-decoration:none;">Pathway enrichment analysis and visualization of omics data using g:Profiler, GSEA, Cytoscape and EnrichmentMap</a></li> 
 
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(37).pdf" style="text-decoration:none;">Normalizing single-cell RNA sequencing data: Challenges and opportunities</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(38).pdf" style="text-decoration:none;">From reads to genes to pathways: differential expression analysis of RNA-Seq experiments using Rsubread and the edgeR quasi-likelihood pipeline [version 2; peer review: 5 approved]</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(39).pdf" style="text-decoration:none;">Single-Cell Transcriptomics Bioinformatics and Computational Challenges</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(40).pdf" style="text-decoration:none;">iRNA-seq: computational method for genome-wide assessment of acute transcriptional regulation from total RNA-seq data</a></li>                              
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(41).pdf" style="text-decoration:none;">Comprehensive evaluation of fusion transcript detection algorithms and a meta-caller to combine top performing methods in paired-end RNA-seq data</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(42).pdf" style="text-decoration:none;">Characterizing noise structure in single-cell RNA-seq distinguishes genuine from technical stochastic allelic expression</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(43).pdf" style="text-decoration:none;">Normalization of RNA-seq data using factor analysis of control genes or samples</a></li>
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(44).pdf" style="text-decoration:none;">HISAT: a fast spliced aligner with low memory requirements</a></li>
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(45).pdf" style="text-decoration:none;">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</a></li>  
   
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(46).pdf" style="text-decoration:none;">Ballgown bridges the gap between transcriptome assembly and expression analysis</a></li> 
                             
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(47).pdf" style="text-decoration:none;">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie, and Ballgown</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(48).pdf" style="text-decoration:none;">The contribution of cell cycle to heterogeneity in single-cell RNA-seq data</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(49).pdf" style="text-decoration:none;">Differential gene and transcript expression analysis of RNA-Seq experiments with TopHat and Cufflinks</a></li>
                              
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(50).pdf" style="text-decoration:none;">Translating RNA sequencing into
clinical diagnostics: opportunities and challenges</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(51).pdf" style="text-decoration:none;">Tackling the widespread and
critical impact of batch effects in high-throughput data</a></li>
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(52).pdf" style="text-decoration:none;">Geometry of the Gene Expression Space of Individual Cells</a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(53).pdf" style="text-decoration:none;">Cross-platform normalization of
microarray and RNA-seq data for machine learning applications</a></li>
 
<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(54).pdf" style="text-decoration:none;">Union Exon Based Approach for RNA-Seq
Gene Quantification: To Be or Not to Be? </a></li>

<li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(55).pdf" style="text-decoration:none;">Reproducibility of high-throughput mRNA and small RNA sequencing across laboratories</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(56).pdf" style="text-decoration:none;">RNA sequencing: the teenage years </a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(57).pdf" style="text-decoration:none;">Errors in RNA-Seq quantification affect genes of relevance to human disease</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(58).pdf" style="text-decoration:none;">Comparing the normalization methods for the differential analysis of Illumina highthroughput RNA-Seq data</a></li>
    <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(59).pdf" style="text-decoration:none;">A comprehensive evaluation of ensembl, RefSeq, and UCSC annotations in the context of RNA-seq read mapping and gene quantification</a></li>
 
  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(60).pdf" style="text-decoration:none;">QuickRNASeq lifts large-scale RNA-seq data analyses to the next level of automation and interactive visualization </a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(61).pdf" style="text-decoration:none;">Tools and best practices for data
processing in allelic expression analysis</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(62).pdf" style="text-decoration:none;">Meta-analysis of RNA-seq expression data across species, tissues and studies</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(63).pdf" style="text-decoration:none;">Isoform prefiltering improves
performance of count-based methods for analysis of differential transcript usage</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(64).pdf" style="text-decoration:none;">A survey of best practices for RNA-seq data analysis</a></li>
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(65).pdf" style="text-decoration:none;">Design and computational analysis of single-cell RNA-sequencing experiments </a></li> 

   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(66).pdf" style="text-decoration:none;">A benchmark for RNA-seq quantification pipelines</a></li> 
 
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(67).pdf" style="text-decoration:none;">Comprehensive analyses of tumor
immunity: implications for cancer immunotherapy</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(68).pdf" style="text-decoration:none;">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods</a></li> 
 
  
   <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(69).pdf" style="text-decoration:none;">A practical guide to single-cell RNAsequencing for biomedical research and clinical applications</a></li>                              

  <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(70).pdf" style="text-decoration:none;">Comparative assessment of
methods for the fusion transcripts detection from RNA-Seq data</a></li> 
  
 
 <li><a target="_blank" href="https://github.com/manjunath5496/RNA-Seq-Analysis-Papers/blob/master/r(71).pdf" style="text-decoration:none;">voom: precision weights unlock linear model analysis tools for RNA-seq read counts</a></li>
 
 </ul>
